ibft/sync/history.go

```diff
@@ -16,23 +17,19 @@ import (
 // fetching decided messages from the network
 type HistorySync struct {
 	logger              *zap.Logger
+	publicKey           *bls.PublicKey
 	network             network.Network
 	ibftStorage         collections.Iibft
 	validateDecidedMsgF func(msg *proto.SignedMessage) error
-	validatorPK         []byte
```

Comment:
@nivBlox why change name or have validatorPK at all?

Reply:
for the get topic